apps/hash-api/src/graph/ontology/primitive/data-type.ts

```diff
@@ -54,7 +51,13 @@ export const createDataType: ImpureGraphFunction<
     kind: "data-type",
     title: params.schema.title,
   });
-  const schema = { $id: dataTypeUrl, ...params.schema };
+  const schema = {
+    $schema:
+      "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type" as const,
```

Comment:
Can you replace these with imports of `DATA_TYPE_META_SCHEMA` and such from the exports of the `@blockprotocol/graph` or `@blockprotocol/type-system` package (preferably graph if it's already a dependency)

Reply:
It only suggested imports from `type-system`: https://github.com/hashintel/hash/pull/2129/commits/0241c04ec2c6616e744b9b2382a31e18fd9a3e44

Reply:
I think it probably did that because there was already an import from the `type-system`. It'd be good to unify all of our imports at some point so we're not importing from both packages, no worries though